core/src/test/java/org/jdbi/v3/TestJsr310.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+public class TestJsr310 {
+    @Rule
+    public H2DatabaseRule db = new H2DatabaseRule();
+
+    Handle h;
+
+    @Before
+    public void setUp() {
+        h = db.getSharedHandle();
+        h.execute("create table stuff (ts timestamp, d date)");
```

Comment:
Per http://mail.openjdk.java.net/pipermail/jdbc-spec-discuss/2012-December/000010.html it looks like the intention was that `ResultSet.setObject` would support some java.time types.

However I'm concerned it may not be uniformly implemented across all drivers. I'll do some digging.